### ISSUE TITLE
Funtion for directly input sympy expression.

### ISSUE THREAD

**Issue by stevenleeS0ht** [sympy/sympy#16467]
In order to input a long expression in string. I have to use function `simplify` or `expand`, but both of them will vary the original expression more or less.

If it is possible, adding a function like `feed` or `take` or `expression` which can keep the original format and convert the string to sympy expression would be awesome.

**Comment by smichr** [comment]
Does `S('''my long expression''')` work for you?

**Comment by stevenleeS0ht** [comment]
Yes, it works perfect, but without reading the official documentation it is hard to find out. @smichr , thanks so much.
